src/components/survey/index.js

```diff
@@ -125,6 +138,14 @@ class Survey extends Component {
               question={q}
             />
           ))}
+          {explanations && explanations.length ? (
+            <Panel>
+              <Explanation
+                className={style.explanation}
+                explanations={explanations}
+              />
+            </Panel>
+          ) : null}
```

Comment:
This can be written as:

```js
        {explanations && explanations.length && (
          <Panel>
            <Explanation
              className={style.explanation}
              explanations={explanations}
            />
          </Panel>
        )}
```